src/scene/forward-renderer.js

```diff
@@ -823,6 +828,143 @@ class ForwardRenderer {
         }
     }
 
+    // placeholder LUT textures for area light
+    _createAreaLightPlaceholderLuts() {
+        var placeholderLutTex =  new Texture(this.device, {
+            width: 2,
+            height: 2,
+            format: PIXELFORMAT_R8_G8_B8_A8
+        });
+        placeholderLutTex.name = 'placeholder';
+
+        var pixels = placeholderLutTex.lock();
+        for (var i = 0; i < 4; i++) {
+            for (var c = 0; c < 4; c++) {
+                pixels[i * 4 + c] = 0;
+            }
+        }
+        placeholderLutTex.unlock();
```

Comment:
We could make this simpler and create a single pixel texture. Maybe even create this function in a way we can reuse it, as I think you copied it from elsewhere? It seems not intuitive to create 2x2 texture and then do 4x4 loop .. even though the result is correct.

copied from Editor, would need to be cleaned up
```
    var createSolidTex = function (name, r, g, b, a) {
        var result = new pc.Texture(app.graphicsDevice, { width: 1, height: 1, format: pc.PIXELFORMAT_R8_G8_B8_A8 });
        result.name = name;
        var pixels = result.lock();
        pixels.set(new Uint8Array([r, g, b, a]));
        result.unlock();
        return result;
    };

```

Reply:
I think this is a good idea - I would prefer that it be in another PR that would unify the other places that uses something similar - something that I also think could be useful is to be able to specify a default/placeholder texture that isn't a solid color and is easily visually identifiable as a missing texture - this can be very useful for debugging.

Reply:
sounds good!

Reply:
placeholder issue for further discussion here: https://github.com/playcanvas/engine/issues/2794